src/utils/StringBuilder/StringBuilder.tsx

```diff
@@ -0,0 +1,27 @@
+import { ROOT_ENDPOINT, BEERS, PAGE_NUMBER, PER_PAGE, BEER_NAME } from '../../shared/data/data';
+
+function buildQueryString(
+  searchString: string | null,
+  pageNumber: number,
+  perPage: number
+): string {
+  let queryString: string;
+
+  if (searchString) {
+    queryString =
+      ROOT_ENDPOINT +
+      BEERS +
+      PAGE_NUMBER +
+      pageNumber +
+      PER_PAGE +
+      perPage +
+      BEER_NAME +
+      searchString.trim();
+  } else {
+    queryString = ROOT_ENDPOINT + BEERS + PAGE_NUMBER + pageNumber + PER_PAGE + perPage;

```

Comment:
you use the same code twice 
![image](https://github.com/ylakhtin/rs-react-components/assets/56221219/940d44d1-a973-4d6a-bd53-ce7b6f105eb1)